ckanext/dataset_subscriptions/helpers.py

```diff
@@ -1,12 +1,21 @@
 from ckan.plugins import toolkit
 import datetime
+import logging
+
+
+logger = logging.getLogger(__name__)
 
 
 def add_dataset_details_to_activity_list(activity_list, context):
     for index, activity in enumerate(activity_list):
         object_id = activity['object_id']
-        dataset = toolkit.get_action('package_show')(
-                    context, {'id': object_id})
+        try:
+            dataset = toolkit.get_action('package_show')(
+                        context, {'id': object_id})
+        except:
+            logger.exception("Unable to get details of package: " + object_id,
```

Comment:
That's an odd line break here. But I don't think you need it anyway:
1. you can use fstrings here `f"Unable to get details of package: {object_id}"` - 0 chars saved
2. with `logging.exception` exc_info is added automatically, you don't need `exc_info=True` https://docs.python.org/3/library/logging.html#logging.Logger.exception -  13 chars saved
Hopefully, this will fit in a single line afterwards.